notebooks/sparse_embedding_retrieval.ipynb

```diff
@@ -0,0 +1,10657 @@
+{
```

Comment:
<p>Is it Fastembed or FastEmbed? The integration page is called 'FastEmbed'</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/79/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
right!
I fixed this.